latex-documents/practical_framework/src/sprint-1.tex

```diff
@@ -0,0 +1,95 @@
+
+\subsection{Sprint 1}
+
+\subsubsection{Introduction}
+
+In this sprint, our focus was on starting the contract for the developer and 
+keep investigation about execution for the game using the console we define.
+That's why we started with UMLs for our first game (Tic Tac Toe), console and
+our contract (frontend-library), that is the Bridge.
+At the same, we had been working on some Spikes for the console, such as to
+define a graphic library for it, communication via JNI, apply shared memory 
+between a game and console and finally, register input events, like mouse and keyboard.
+All these Skipes were related in the context of the technologies used in the project.
+
+\subsubsection{UMLs}
+
+We had been starting on the UMLs for the contract, Tic Tac Toe game and console.
+
+\begin{itemize}
+    \item Task ID - 17: \href{https://tree.taiga.io/project/joseluis-teran-coffeetime/us/17?milestone=392128}{Console Diagram on Taiga (Comments Section)}
+    \item Task ID - 18: \href{https://github.com/Pending-Name-21/arquitecture/pull/2}{Library UML Diagram on GitHub}
+    \item Task ID - 22: \href{https://github.com/Pending-Name-21/arquitecture/pull/3}{Tic Tac Toe UML Diagram on GitHub}
+\end{itemize}
+
+\subsubsection{Spikes}
+
+As we mentioned before, there were some Spikes to work with and here is the documentation generated by them:
+
+\begin{itemize}    
+    \item Task ID - 19: \href{https://docs.google.com/document/d/1a6wyQA0LM5thyAfOsWnkG-fTLcThvrsVPOGmFcoqkvw/edit?usp=sharing}{Graphic Libraries Documentation on Google Docs}
+    \item Task ID - 20: \href{https://github.com/Pending-Name-21/console/tree/vm-spikes/jni_spike}{JNI Documentation on GitHub}
+    \item Task ID - 21: \href{https://github.com/Pending-Name-21/console/blob/vm-spikes/shared-memory/README.md}{Shared Memory Documentation on GitHub}
+    \item Task ID - 24: \href{https://universidadsalesian-my.sharepoint.com/:w:/g/personal/axel_ayala_9412013_usalesiana_edu_bo/EZlHobuXqW5AmffmDNnGaKYBdpordz1QlVJk88Pe_6S7HQ?e=DymfMq}{Input Listener Documentation on Word}
+\end{itemize}
+
+\subsubsection{POCs}
+
+In this Sprint, we had no Proofs of Concept (POCs).
+
+\subsubsection{Technical Justification}
+
+During Sprint 1, the team exclusively focused on designing UMLs and
+generating Spikes for console. This decision was based on the following reasons:
+
+\subsubsection{High - Level Architecture}
+
+The architecture followed in this sprint was based on the initial architecture.
+
+\begin{itemize}
+    \item Architecture: \href{https://github.com/Pending-Name-21/arquitecture/pull/1/files}{Architecture Initial Version}
+    You can see it on: 
+    \item C4 DSL: \href{https://structurizr.com/dsl}{Visualizer}
+\end{itemize}
+
+\newpage
+
+\subsubsection{Epics}
+
+For this Sprint 1, we have the following epics:
+
+\begin{itemize}
+    \item Bridge
+    \item Console
+    \item Graphics
+    \item Games
+\end{itemize}
+
+\subsubsection{Sprint Backlog}
+
+Completed Spikes and User Stories.
```

Comment:
The backlogs file was already merged, maybe you could just import it here.

Reply:
Okay! I'll add it when the rest of artifacts are done.